packages/netlify-cms-backend-github/src/implementation.js

```diff
@@ -221,30 +221,30 @@ export default class GitHub {
   }
 
   async entriesByFolder(collection, extension) {
-    const repoURL = `/repos/${this.useOpenAuthoring ? this.originRepo : this.repo}`;
+    const repoURL = this.useOpenAuthoring ? this.api.originRepoURL : this.api.repoURL;
     const files = await this.api.listFiles(collection.get('folder'), { repoURL });
     const filteredFiles = files.filter(file => file.name.endsWith('.' + extension));
-    return this.fetchFiles(filteredFiles, { repoURL });
+    return this.fetchFiles(filteredFiles);
   }
 
   entriesByFiles(collection) {
-    const repoURL = `/repos/${this.useOpenAuthoring ? this.originRepo : this.repo}`;
+    const repoURL = this.useOpenAuthoring ? this.api.originRepoURL : this.api.repoURL;
     const files = collection.get('files').map(collectionFile => ({
       path: collectionFile.get('file'),
       label: collectionFile.get('label'),
     }));
-    return this.fetchFiles(files, { repoURL });
+    return this.fetchFiles(files);
   }
 
-  fetchFiles = (files, { repoURL = `/repos/${this.repo}` } = {}) => {
+  fetchFiles = (files, { repoURL = this.api.repoURL } = {}) => {
     const sem = semaphore(MAX_CONCURRENT_DOWNLOADS);
     const promises = [];
     files.forEach(file => {
       promises.push(
         new Promise(resolve =>
           sem.take(() =>
             this.api
-              .readFile(file.path, file.sha, { repoURL })
+              .readFile(file.path, file.sha)
```

Comment:
This should be `.readFile(file.path, file.sha, { repoURL })`

Reply:
Added those changes and retested on my end